### PR TITLE
Add missing description and fix typos for Config-Quick-Reference

### PR DESCRIPTION
**Pull request by big-r81**

### CODE REVIEW

File: src/config/http.rst [hunk 1]
```diff
@@ -129,7 +129,7 @@ HTTP Server Options
             [chttpd]
             changes_timeout = 60000 ; 60 seconds
 
-    .. config:option:: config_whitelist :: Config options while list
+    .. config:option:: config_whitelist :: Config options white list
```
Comment (flimzy) [inline src/config/http.rst:132]: I think it should also be a single word, to be consistent with the following paragraph.

```suggestion
    .. config:option:: config_whitelist :: Config options whitelist
```

Comment (flimzy) [reply src/config/http.rst:132]: I'm also a bit surprised to see the word `whitelist` still used within CouchDB. We made an effort to remove certain words in the past, but it looks like this one didn't get caught.

Comment (big-r81) [reply src/config/http.rst]: Yes, you are right. Should we rename the config option to e.g.: config_allowlist or config_acceptlist or any other better name?

Comment (flimzy) [reply src/config/http.rst]: IMHO, either is fine. I'm happy to defer to anyone who has an actual preference :)

Comment (big-r81) [reply src/config/http.rst]: Okay, thats a bigger discussion not only for backwards compatibility when changing config options. I will write a post on slack for further opinions, but good catch!

**Review by flimzy — APPROVED** [review]

**Comment by flimzy** [comment]
> Not ready yet, more to come ;-)

:see_no_evil:   Sorry for jumping the gun :grin:

File: src/config/replicator.rst [hunk 2]
```diff
@@ -283,15 +285,25 @@ Fair Share Replicator Share Allocation
...
 
-        Fair share configuration section. More shares result in a
+        Fair share configuration section. Higher share values results in a
         higher chance that jobs from that db get to run. The default
         value is 100, minimum is 1 and maximum is 1000. The
-        configuration may be set even if the database does not exist::
+        configuration may be set even if the database does not exist.
+
+        Please adjust the variable ``$replicator_db`` to your replicator
+        database name. The default replicator database is _replicator.
```
Comment (nickva) [inline src/config/replicator.rst:298]: Tiny nit: ``` ``_replicator``  ``` so the style matches with ``` ``$replicator_db`` ```?

Comment (big-r81) [reply src/config/replicator.rst]: Yes, looks better.

File: src/config/replicator.rst [hunk 3]
```diff
@@ -283,15 +285,25 @@ Fair Share Replicator Share Allocation
...
+        Additional replicator databases can be created. To be recognized
+        as such by the system, their database names should end with
+        /_replicator. See the :ref:`Replicator Database <replicator>` section
```
[the remark this answers is not in the document]
Comment (big-r81) [reply src/config/replicator.rst]: The same, yes.

**Review by nickva — APPROVED** [review]
+1 with a few tiny nits

File: src/config/logging.rst [hunk 4]
```diff
@@ -106,31 +106,31 @@ Logging options
...
+    .. config:option:: syslog_appid :: Application name for syslog
 
         Specifies application name to the ``syslog``
         :option:`writer <log/writer>`::
 
             [log]
             syslog_appid = couchdb
 
-    .. config:option:: syslog_facility
+    .. config:option:: syslog_facility :: SYSLOG designations for message sources
```
[the remark this answers is not in the document]
Comment (big-r81) [reply src/config/logging.rst]: Changed. Btw, something seems to be wrong with the logging to the facility, see [couchdb#3882](https://github.com/apache/couchdb/issues/3882)

**Comment by big-r81** [comment]
Btw, something seems to be wrong with the logging to the facility, see [couchdb#3882](https://github.com/apache/couchdb/issues/3882)

**Pull request merged**